frontend/test/model/common/ValidationErrorIcon.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { ThemeProvider } from '@mui/material/styles'
+import { render, screen, waitFor } from '@testing-library/react'
+import { MarkdownDisplayProps } from 'src/common/MarkdownDisplay'
+import ValidationErrorIcon from 'src/model/beta/common/ValidationErrorIcon'
+import { lightTheme } from 'src/theme'
+import { testAccessRequestSchemaStepNoRender } from 'utils/test/testModels'
+import { describe, expect, it, vi } from 'vitest'
+
+vi.mock('src/common/MarkdownDisplay.tsx', () => ({ default: (_props: MarkdownDisplayProps) => <></> }))
+
+describe('Va;idationError', () => {
```

Comment:
The describe should probably be the exact name of the component: `ValidationErrorIcon` so that it's easier to debug if needed.